keyboards/divergetm2/readme.md

```diff
@@ -0,0 +1,21 @@
+# diverge tm2
+
+A 4x6x2 split ortholinear keyboard with 2u spacebars like the Levinson (similar to Let's Split). Made by [Unikeyboard](https://unikeyboard.io).
+
+Keyboard Maintainer: [IslandMan93](https://github.com/islandman93) and [xton](https://github.com/xton), I guess?
+Hardware Supported: Pro Micro
+Hardware Availability: [Diverge TM2](https://unikeyboard.io/product/diverge-tm/)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make divergetm2:default
+
+See [build environment setup](https://docs.qmk.fm/build_environment_setup.html) then the [make instructions](https://docs.qmk.fm/make_instructions.html) for more information.
```

Comment:
Could you please replace this line with the following?

`See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).`